README.md

```diff
@@ -1,6 +1,53 @@
-zeplic
-======
+# zeplic 0.1.0
 
 [![Build Status](https://travis-ci.org/IgnacioCarbajoVallejo/zeplic.svg?branch=master)](https://travis-ci.org/IgnacioCarbajoVallejo/zeplic)
 
-ZFS Datasets distribution over datacenter - Let'zeplic 
+ZFS Datasets distribution over datacenter - Let'zeplic
+
+**Tested on FreeBSD**
+
+## Process
+
+1. Get clone dataset
+2. Destroy clone dataset
+3. Get dataset (called in JSON file)
+4. Destroy dataset (disable)
+5. Create dataset if it does not exist
+6. Save the last #Retain(JSON file) snapshots
+7. Create a new snapshot
+8. Create a clone of last snapshot
+9. Rollback of last snapshot (disable)
+
+## Utils
+
+- System logging daemon.
+
+## How can you use it?
+
+First, clone this repository into `$GOPATH/src/github.com/nfrance-conseil/zeplic` and export your `$GOBIN`.
+The next step is to configure **zeplic**:
+
```

Comment:
Makefile is a solution (including a configuration for the syslogd) or maybe, to do a freebsd port collection with an binary package